addons/rose/addon/components/rose/code-editor/toolbar/index.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { later } from '@ember/runloop';
+
+export const COPY_ICON_TYPE = 'flight-icons/svg/clipboard-copy-16';
+export const COPIED_ICON_TYPE = 'flight-icons/svg/clipboard-checked-16';
+
+export default class RoseCodeEditorToolbarComponent extends Component {
+  @tracked copyIconType = COPY_ICON_TYPE;
+
+  @action
```

Comment:
This [used to be a task](https://github.com/hashicorp/cloud-ui/blob/master/packages/cloud-ui-core/addon/components/code-editor/toolbar/index.js#L94) and they originally used ember concurrency.

I noticed we didn't have it installed and it seemed to be in essence a `setTimeout` to transition the button which seemed a bit excessive to install two packages for this. 

Let me know if there's any problems with not using ember-concurrency for this